js/apps/account-ui/src/personal-info/PersonalInfo.tsx

```diff
@@ -79,6 +87,31 @@ const PersonalInfo = () => {
             {t("doCancel")}
           </Button>
         </ActionGroup>
+        {environment.features.deleteAccountAllowed && (
+          <ExpandableSection toggleText={t("deleteAccount")}>
+            <Alert
+              isInline
+              title={t("deleteAccount")}
+              variant="danger"
+              actionLinks={
+                <Button
+                  id="delete-account-btn"
```

Comment:
We're adding an `id` here, but it's never used. Is this to accommodate the existing tests?

Reply:
indeed